extensions/2.0/Khronos/KHR_materials_specular/README.md

```diff
@@ -0,0 +1,247 @@
+# KHR\_materials\_specular
+
+## Contributors
+
+* Tobias Haeussler, Dassault Systemes [@proog128](https://github.com/proog128)
+* Bastian Sdorra, Dassault Systemes [@bsdorra](https://github.com/bsdorra)
+* Ed Mackey, AGI [@emackey](https://twitter.com/emackey)
+* Richard Sahlin, IKEA [@rsahlin](https://github.com/rsahlin)
+* Ben Houston, ThreeKit [@bhouston](https://github.com/bhouston)
+* Emmett Lalish, Google [@elalish](https://github.com/elalish)
+* Don McCurdy, Google [@donrmccurdy](https://twitter.com/donrmccurdy)
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+* Sebastien Vandenberghe, Microsoft [@sebavanjs](https://twitter.com/sebavanjs)
+
+Copyright (C) 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension adds two parameters to the metallic-roughness material: `specular` and `specularColor`.
+
+`specular` allows users to configure the strength of the specular reflection in the dielectric BRDF. A value of zero disables the specular reflection, resulting in a pure diffuse material. The metal BRDF is not affected by the parameter.
+
+`specularColor` changes the F0 color of the specular reflection in the dielectric BRDF, allowing artists to use effects known from the specular-glossiness material (`KHR_materials_pbrSpecularGlossiness`) in the metallic-roughness material.
+
+## Extending Materials
+
+The strength of the specular reflection is defined by adding the `KHR_materials_specular` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_specular": {
+                    "specularFactor": 1.0,
+                    "specularColorFactor": [1.0, 1.0, 1.0],
+                }
+            }
+        }
+    ]
+}
+```
+
+Factor and texture are combined by multiplication to describe a single value.
+
+| |Type|Description|Required|
+|-|----|-----------|--------|
+| **specularFactor** | `number` | The strength of the specular reflection. | No, default: `1.0`|
```

Comment:
Perhaps this should be called just 'strength' to be in line with for instance Occlusion?

Reply:
"Factor" has a particular meaning in glTF, and that's what this is.  It could be `specularStrengthFactor` if need be, but let's discuss with the group before making any further changes.

Reply:
The occlusion one is called strength because it is different than other "factor"s. It's not just a straight multiply. Same goes for normal scale.

https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#occlusiontextureinfostrength
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#normaltextureinfoscale